app/models/viewmodels/notificationspage/converters/MovementResponseConverter.scala

```diff
@@ -51,7 +51,7 @@ class MovementResponseConverter @Inject()(decoder: Decoder, dateTimeFormatter: D
 
   private def buildCrcCodeExplanation(crcCode: String)(implicit messages: Messages): Option[String] = {
     val CrcCodeHeader = messages("notifications.elem.content.inventoryLinkingMovementResponse.crc")
-    decoder.crc(crcCode).map(code => paragraph(s"$CrcCodeHeader ${code.contentKey}"))
+    decoder.crc(crcCode).map(code => paragraph(s"$CrcCodeHeader ${code.messageKey}"))
```

Comment:
missing messages ?

Reply:
This is being fixed in CEDS-1736. I am preparing PR for this today.